chapters/it/chapter5/8.mdx

```diff
@@ -0,0 +1,226 @@
+<!-- DISABLE-FRONTMATTER-SECTIONS -->
+
+# Quiz di fine capitolo
+
+In questo capitolo abbiamo fatto un bel po' di strada! Non preoccuparti se non hai colto tutti i dettagli; i capitoli successivi ti aiuteranno a capire come funzionano le cose dietro le quinte!
+
+Prima di andare oltre, mettiamo alla prova ciò che hai imparato in questo capitolo.
+
+### 1. Usando la funzione `load_dataset()` in 🤗 Datasets, da dove puoi caricare un dataset?
+
+<Question
+	choices={[
+		{
+			text: "In locale, ad esempio dal tuo portatile.",
+			explain: "Corretto! Puoi passare i path dei file nell'argomento <code>data_files</code> di <code>load_dataset()</code> per caricare dataset locali.",
+			correct: true
+		},
+		{
+			text: "L'Hub Hugging Face.",
+			explain: "Corretto! Puoi caricare i dataset presenti sull'Hub fornendo l'ID del dataset, ad esempio <code>load_dataset('emotion')</code>."
```

Comment:
Looking at the logs I think this is the culprit:

```suggestion
			explain: "Corretto! Puoi caricare i dataset presenti sull'Hub fornendo l'ID del dataset, ad esempio <code>load_dataset('emotion')</code>.",
```

Reply:
Thanks a lot for the help!